io.github.milkshiift.GoofCord.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <!--Created with jdAppdataEdit 5.1-->
+  <id>io.github.milkshiift.GoofCord</id>
+  <name>GoofCord</name>
+  <summary>Take control of your Discord experience with GoofCord – the highly configurable and privacy minded discord client.</summary>
+  <developer_name>MilkShift</developer_name>
+  <launchable type="desktop-id">io.github.milkshiift.GoofCord.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>OSL-3.0</project_license>
+  <project_group>GoofCord</project_group>
+  <description>
+    <ul>
+      <li><b>🔒 With Privacy in mind:</b> GoofCord blocks all tracking and uses multiple techniques like message encryption to improve your privacy and security. <a href="https://github.com/Milkshiift/GoofCord/wiki/Privacy-FAQ">Learn more</a></li>
+      <li><b>⚙️ Your Way, Your Client:</b> Set up GoofCord to your liking with an array of customization options.</li>
+      <li><b>📈 Fast and Performant:</b> Glide through your chats with GoofCord's superior speed and responsiveness compared to the official client.</li>
+      <li><b>🔖 Standalone:</b> GoofCord is a standalone client, not reliant on the original Discord client in any way.</li>
+      <li><b>🔌 Plugins &amp; Themes support:</b> Use client mods like <a href="https://github.com/Vendicated/Vencord">Vencord</a> or <a href="https://github.com/uwu/shelter">Shelter</a> for a world of plugins and themes.</li>
+      <li><b>🐧 Linux support:</b> Seamless screen sharing with audio and native Wayland support on Linux. See <a href="https://github.com/Milkshiift/GoofCord/#faq">FAQ</a> for details.</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>GoofCord with settings open on a themed Kubuntu</caption>
+      <image type="source" width="1920" height="979">https://raw.githubusercontent.com/Milkshiift/GoofCord/main/assetsDev/screenshot1.png</image>
```

Comment:
this should NOT link to branch but to a specific release as to not break.

Reply:
metainfo now uses "https://raw.githubusercontent.com/Milkshiift/GoofCord/939c90f0b8d10d551900cc1aea0458d8a36b45c8/assetsDev/screenshot1.png" which should be permanent